book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+    rental.book = self
+  end
```

Comment:
- [ ] Your add_rental method should look something like below ⬇️ to make many-to-many association 👍
```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```